### PR TITLE
Added object - style API, custom field name for properties table and fixed namespace

### PR DESCRIPTION
**Pull request by ruslanbat**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
-    ],
-    "minimum-stability": "dev",
+    ], 
     "require": {
 
     },
     "autoload": {
         "psr-4": {
-            "asdfstudio\\eav\\": ""
+            "app\\components\\": ""
```
Comment (asdf404) [inline composer.json]: Why you changed namespace and package name?

File: EavBehavior.php [hunk 2]
```diff
@@ -48,14 +63,14 @@ public function events()
      */
     public function afterFind()
     {
-        $properties = [];
+        $properties = (object) [];
```
Comment (asdf404) [inline EavBehavior.php]: It breaking change of API

Comment (ruslanbat) [inline EavBehavior.php]: When  use old version  cant set value as $model->properties['foo'] = 'foo';. I got 'Notice: Indirect modification of overloaded property MyClass::$properties has no effect'.  maybe add option a data as object

Comment (asdf404) [inline EavBehavior.php]: @ruslanbat maybe implement of [ArrayAccess](http://php.net/manual/ru/class.arrayaccess.php) will be best solution?

Comment (ruslanbat) [inline EavBehavior.php]: @asdf404 so ArrayAccess is optimal

File: composer.json (hunk 1 above)
Comment (ruslanbat) [inline composer.json]: namespace changed, because I think they should be similar to the pattern YII
Package name back to old version

Comment (asdf404) [inline composer.json]: @ruslanbat if someone installs two (or more) packages with same namespaces then project will blows up. Anyway it breaks existing code.

**Pull request closed without merging**